flixel/system/FlxSound.hx

```diff
@@ -736,17 +735,24 @@ class FlxSound extends FlxBasic
 		updateTransform();
 		return Volume;
 	}
-
-	#if (sys && openfl_legacy)
+	#if FLX_PITCH
 	inline function get_pitch():Float
 	{
 		return _pitch;
 	}
+	
 
 	function set_pitch(v:Float):Float
 	{
 		if (_channel != null)
+			#if openfl_legacy
 			_channel.pitch = v;
+			#else
+			@:privateAccess
+			if (_channel.__source != null)
+				_channel.__source.pitch = v;
```

Comment:
getting errors here:
> .../FlxSound.hx:752: characters 17-25 : flash.media.SoundChannel has no field __source
.../FlxSound.hx:753: characters 14-22 : flash.media.SoundChannel has no field __source

There must be a problem with the FLX_PITCH definition